### PR TITLE
fix bugs for submit order draft

### PR DESCRIPTION
**Pull request by Anstsiya**
[AB#57560](https://dev.azure.com/orckestra001/da965986-c85b-43be-9915-219568b000e5/_workitems/edit/57560)

### CODE REVIEW

File: src/Orckestra.Composer.Website/App_Data/Razor/Composer/Cart/FullCart.cshtml [hunk 1]
```diff
@@ -30,7 +30,7 @@
                     <div class="col-md-6  col-md-push-6  col-lg-push-0  col-lg-12">
                         @Function("Composer.Cart.OrderSummary")
                     </div>
-                    <div class="col-md-6  col-md-pull-6  col-lg-pull-0  col-lg-12">
+                    <div v-if="Cart.CartType == '!@CartConfiguration.OrderDraftCartType'" class="col-md-6  col-md-pull-6  col-lg-pull-0  col-lg-12">
```
Comment (InnaBoitsun) [inline src/Orckestra.Composer.Website/App_Data/Razor/Composer/Cart/FullCart.cshtml:33]: does it work?

File: src/Orckestra.Composer.Cart/Providers/Order/EditingOrderProvider.cs [hunk 2]
```diff
@@ -253,18 +253,40 @@ private Task<ProcessedCart> ChangeOwnership(Overture.ServiceModel.Orders.Order o
 
         public virtual async Task<Overture.ServiceModel.Orders.Order> SaveEditedOrderAsync(Overture.ServiceModel.Orders.Order order)
         {
-            var updatedOrder = await OrderRepository.SubmitCartOrderDraftAsync(new SubmitCartOrderDraftParam
+            var updatedOrder = new Overture.ServiceModel.Orders.Order();
```
Comment (InnaBoitsun) [inline src/Orckestra.Composer.Cart/Providers/Order/EditingOrderProvider.cs:256]: use better this: Overture.ServiceModel.Orders.Order updatedOrder;

File: src/Orckestra.Composer.Cart/Providers/Order/EditingOrderProvider.cs [hunk 3]
```diff
@@ -253,18 +253,40 @@ private Task<ProcessedCart> ChangeOwnership(Overture.ServiceModel.Orders.Order o
...
-            }).ConfigureAwait(false);
+                updatedOrder = await SaveEditedOrder(order);
+            }
+            catch (ComposerException ex)
+            {
+                var ownedBySomeoneElseError = ex.Errors?.FirstOrDefault(e => e.ErrorCode == Constants.ErrorCodes.IsOwnedBySomeoneElse);
+                if (ownedBySomeoneElseError != null)
+                {
+                    await ChangeOwnership(order).ConfigureAwait(false);
+                    updatedOrder = await SaveEditedOrder(order);
```
Comment (InnaBoitsun) [inline src/Orckestra.Composer.Cart/Providers/Order/EditingOrderProvider.cs:267]: add .ConfigureAwait(false) at the end

**Review by InnaBoitsun — CHANGES_REQUESTED** [review]

File: src/Orckestra.Composer.Website/App_Data/Razor/Composer/Cart/FullCart.cshtml (hunk 1 above)
Comment (Anstsiya) [reply src/Orckestra.Composer.Website/App_Data/Razor/Composer/Cart/FullCart.cshtml]: fixed, now it works

**Review by InnaBoitsun — APPROVED** [review]

**Pull request merged**